cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -77,7 +77,7 @@ ELSEIF ("$ENV{ATDM_CONFIG_BUILD_TYPE}" STREQUAL "DEBUG")
 ELSE()
   MESSAGE(FATAL_ERROR
     "Error, env var ATDM_CONFIG_BUILD_TYPE=$ENV{ATDM_CONFIG_BUILD_TYPE} is"
-    " set to an illegal value!  Only the values 'RELEASE_DEBUG', 'RELEASE',"
+    " set to an illegal value!  Only the values 'RELEASE-DEBUG', 'RELEASE',"
```

Comment:
How come everything else became an underscore and only this one became a hyphen?

Reply:
@bartlettroscoe makes sense -- thanks for explaining!